otx/algorithms/segmentation/configs/base/models/semi_mt_segmentor.py

```diff
@@ -0,0 +1,10 @@
+"""Segmentor config for semi-supervised learning."""
+
+model = dict(
+    type="MeanTeacher",
+    orig_type="EncoderDecoder",
+    unsup_weight=0.1,
+    train_cfg=dict(mix_loss=dict(enable=False, weight=0.1)),
```

Comment:
As far as I know, `mix_loss` is designed for OCR head.. :( 
Could you verify my opinion and remove if I'm right?

Reply:
It was not designed for OCR. It was using inside each head and it was just specified in cfg to make sure to disable mix_loss for test.